src/database/migrations/postgres/20240314140913_accounts_table.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TYPE account_origin AS ENUM ('cli', 'github');
+
+CREATE TABLE accounts(
+    id uuid NOT NULL,
+    PRIMARY KEY (id),
+    email TEXT NOT NULL UNIQUE,
+    account_name TEXT NOT NULL,
+    display_name TEXT NOT NULL,
```

Comment:
let's pick sensible limits and use `varchar` here too

Reply:
Changed